lib/game.rb

```diff
@@ -26,10 +32,10 @@ def player_turn
   end
 
   def grid_view
-    @grid_view ||= GridView.new(output: output)
+    GridView.new(output: output, grid: grid)
```

Comment:
Mostly removed the memoizing because I felt anxious about mutations and the state of the grid. I don't think this has a functional effect on anything.

Reply:
Yeah, memoization can cause weird issues. And if the class caches the template in some way, we don't have to worry at all about the latency of loading that file all the time.

Reply:
Also, memoization with the insecurity of the `@grid` state would definitely cause trouble. Holding state exclusively in `@grid` instead of resetting it would probably allow memoization. It isn't necessary though since there isn't a performance penalty association with making and running the class.